stress/src/user_events_enabled.rs

```diff
@@ -0,0 +1,38 @@
+// To run the test, execute the following command in the stress directory as sudo:
+// sudo -E ~/.cargo/bin/cargo rnu --bin user-events-enabled --release
+
+// TODO : Add stess result here.
+
+mod throughput;
+use eventheader_dynamic::{Provider, ProviderOptions};
+use lazy_static::lazy_static;
+
+// Global constants for level and keyword
+const LEVEL: u8 = 4; // Example level (Informational)
```

Comment:
couple of suggestions
1. Lets add one test with and without tracepoint being enabled.
2. Instead of testing just the event header, lets test via OTel itself, so we know the overall cost/contention, not just that of EventHeader crate.

Reply:
> Lets add one test with and without tracepoint being enabled.

Yes, I thought about it. This needs some thinking, as there needs to be some listener for the tracepoint. Can we do it as separate PR?

> Instead of testing just the event header, lets test via OTel itself, so we know the overall cost/contention, not just that of EventHeader crate.

This test was specifically for cost incurred for `eventheader` crate. Plan is to add separate test for the actual export and enabled check through Otel.

Reply:
if this is only testing event header (that should be done by that repo!), lets rename to "eventheader_enabled" / similar.